packages/tasks/src/model-libraries.ts

```diff
@@ -933,6 +933,14 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: true,
 		countDownloads: `path:"pytorch_model.bin" OR path:"model.safetensors"`,
 	},
+	torchgeo: {
```

Comment:
There are currently no models under https://huggingface.co/models?other=torchgeo. You should tag related models by adding `library_name: torchgeo` in their model card metadata. We usually don't merge until at least 1 or more models are made available.

Reply:
Thanks for catching this. I've updated all of our model cards.

Reply:
Great! 23 models in https://huggingface.co/models?other=torchgeo so far :tada:

Reply:
23 repos, ~90 models. Eventually we'll fix that.